### PR TITLE
Adding AVS Release Considerations

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: docs/eigenlayer/avs-guides/avs-developer-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 6
+title: AVS Developer Best Practices
+---
+
+## AVS Developer Best Practices
+
+Note that these best practices should be considered by operators when they’re considering running the AVS software. EigenLabs will not do any checking or scanning of on chain or off chain software.
```
Comment (antojoseph) [inline docs/eigenlayer/avs-guides/avs-developer-best-practices.md:8]: rephrase to ` EigenLabs is not responsible for validating on-chain or off-chain software`

**Review by antojoseph — APPROVED** [review]
lgtm, made one comment

File: docs/eigenlayer/avs-guides/avs-developer-best-practices.md (hunk 1 above)
Comment (wesfloyd) [reply docs/eigenlayer/avs-guides/avs-developer-best-practices.md:8]: Good call, done

File: docs/eigenlayer/avs-guides/avs-developer-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 6
+title: AVS Developer Security Best Practices
+---
+
+## AVS Developer Security Best Practices
+
+Note that these best practices should be considered by operators when they’re considering running the AVS software. EigenLabs is not responsible for validating on-chain or off-chain software.
```
Comment (NimaVaziri) [inline docs/eigenlayer/avs-guides/avs-developer-best-practices.md:8]:
```suggestion
```

**Review by NimaVaziri — APPROVED** [review]

**Pull request merged**